tests/samples/test_api.py

```diff
@@ -376,7 +376,11 @@ async def test(
         assert resp.headers["Location"] == snapshot
         assert await resp.json() == snapshot
 
-        assert await client.db.samples.find_one() == snapshot
+        test_sample = await client.db.samples.find_one()
+
+        test_sample.pop("job")
+
+        assert test_sample == snapshot
```

Comment:
I would update the snapshot instead of mutating the returned data. Doing it like this means we don't get testing coverage for the job field.